src/main/java/appeng/container/implementations/ContainerCraftingCPU.java

```diff
@@ -133,6 +137,7 @@ protected void setCPU( ICraftingCPU c )
 				this.list.resetStatus();
 				this.monitor.getListOfItem( this.list, CraftingItemList.ALL );
 				this.monitor.addListener( this, null );
+				this.eta = 0;
 			}
```

Comment:
can simplify this block due to `null` check in `instanceof`. That way `this.monitor` is never `null`